exporters/otlp/metrics/src/main/java/io/opentelemetry/exporter/otlp/metrics/OtlpGrpcMetricExporter.java

```diff
@@ -78,7 +78,19 @@ public void onSuccess(@Nullable ExportMetricsServiceResponse response) {
 
           @Override
           public void onFailure(Throwable t) {
-            logger.log(Level.WARNING, "Failed to export metrics", t);
+            if (t.getMessage().contains("UNIMPLEMENTED")) {
```

Comment:
I think you can use `Status.fromThrowable` for more structured access. Is it also a good time to add a message for UNAVAILABLE, the most common exception we get asked about now adays?

Reply:
when does UNAVAILABLE happen? Very happy to add it, if I know a good message to provide.

Reply:
It's generally a connectivity issue. The collector being down, the endpoint being set wrong, network config blocking the requests. A message along the lines of making sure collector is running and reachable by network

Reply:
ok, check it out now.